modules/vaos/app/controllers/vaos/v2/providers_controller.rb

```diff
@@ -3,9 +3,16 @@
 module VAOS
   module V2
     class ProvidersController < VAOS::BaseController
+      STATSD_KEY = 'api.vaos.providers'
+
       def show
         provider_data = vaos_serializer.serialize(provider, 'providers')
         render json: { data: provider_data }
+      rescue => e
```

Comment:
This is only going to log a metric on failure, we might want to log it on success as well.